tests/test_mathematical.py

```diff
@@ -976,56 +976,71 @@ def test_hypot(self, dtype):
         dp_array1 = dpnp.array(array1_data, dtype=dtype)
         dp_array2 = dpnp.array(array2_data, dtype=dtype)
         dp_out = dpnp.array(out, dtype=dtype)
-        result = dpnp.hypot(dp_array1, dp_array2, out=dp_out)
+        result = dpnp.divide(dp_array1, dp_array2, out=dp_out)
 
         # original
         np_array1 = numpy.array(array1_data, dtype=dtype)
         np_array2 = numpy.array(array2_data, dtype=dtype)
-        expected = numpy.hypot(np_array1, np_array2, out=out)
+        expected = numpy.divide(np_array1, np_array2, out=out)
 
-        assert_allclose(expected, result)
-        assert_allclose(out, dp_out)
+        tol = 1e-07
+        assert_allclose(expected, result, rtol=tol, atol=tol)
```

Comment:
we can use a helper function here, like
```suggestion
        assert_dtype_allclose(result, expected)
```